core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -26,8 +26,10 @@ oppia.directive('simpleEditorSidebar', [
       controller: [
         '$scope', 'EditorModeService', 'SimpleEditorManagerService',
         'ScrollSyncService', 'QuestionIdService',
+        'SimpleEditorSidebarModeService',
         function($scope, EditorModeService, SimpleEditorManagerService,
-                 ScrollSyncService, QuestionIdService) {
+                 ScrollSyncService, QuestionIdService,
+                  SimpleEditorSidebarModeService) {
```

Comment:
Oh Sorry I have not pushed my commits yet, please wait for some time. Also, I will de-indent by 1.